### PR TITLE
Added LevelContinuum and tests

### PR DESCRIPTION
**Pull request by whaeck**
The continuum levels. LevelContinuum also has the possibility of specifying a range of levels included in the continuum, e.g. the continuum including levels 10 to 15: [continuum:e10-e15]

### CODE REVIEW

File: src/elementary/LevelContinuum.hpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#include "elementary/LevelNumber.hpp"
+
+namespace njoy {
+namespace elementary {
+
+  /**
+   *  @class
+   *  @brief The nuclear level continuum
+   */
+  class LevelContinuum {
```
Comment (jlconlin) [inline src/elementary/LevelContinuum.hpp:19]: This name seems a little odd to me. Would it make sense to make a `level` namespace?

Comment (whaeck) [reply src/elementary/LevelContinuum.hpp:19]: I'm actually considering of putting this in the LevelNumber class (and renaming LevelNumber to Level). The continuum range of levels is currently unused so I don't really need the second level number anyway. Once we do need it, we can always redesign things.

Comment (jlconlin) [reply src/elementary/LevelContinuum.hpp:19]: I don't want to spend a lot of time doing redesign. We are kind of at a time crunch right now.

Comment (whaeck) [reply src/elementary/LevelContinuum.hpp:19]: I'm talking about a redesign later, when we need a continuum between given levels

**Pull request closed without merging**